### DISCUSSION TITLE
Quarkus 3.2.0.Final released - New security features, @QuarkusComponentTest

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#34558] [Quarkus Blog/Website]
# Quarkus 3.2.0.Final released - New security features, @QuarkusComponentTest

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-3-2-0-final-released/

**Comment by adampoplawski** [comment]
Hello
Is this first LTS version as mentioned in previous blogs?

**Reply by maxandersen** [reply]
Yes, see https://quarkus.io/blog/lts-releases/

**Comment by chiradip** [comment]
Congratulations guys. Keep up the good work.
